static/js/containers/ProfileFormContainer.js

```diff
@@ -98,7 +98,13 @@ class ProfileFormContainer extends React.Component {
         dispatch(clearProfileEdit(username));
       });
     } else {
-      return Promise.reject(errors);
+      return Promise.reject(errors).catch(() => {
```

Comment:
Can we replace this `else` block with everything inside the `catch` here? This line seems redundant